### ISSUE TITLE
The BOTTOM bar of RS, for first user experience.

### ISSUE THREAD

**Issue by felisucoibi** [RetroShare/RetroShare#614]
I would HIDE almost everything from bottom bar of RS.

The reason-guideline what is really important for a NEW user to see all the time everywhere and what no.

In options > Appearence you can enable them again.

Make this question, is important for me as a RS USER this information or this menu ALL THE TIME? 
HOW MANY TIMES I USED THAT INFORMATION? I also used some new user information to realize they should not be there at the first time.

1. The ONLINE, BUSY stuff... first of all, between online and away, it can be done automatically, if you are away x seconds change to away... Why tell people you are busy there?
Is the best place there? Or maybe is something that should be in chats? or private chats? I mean I'm downloading files and I have the online away option I'm in a forum and the same... should be placed inside the parts of RS that really needs that... chat, private messaging... 
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE

2. On line friends... is something we really need to know all the time? There is a News Feed that says if your friend is on line all the time, there is a Network place where you can set it up to show only on line friends....
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE

2.5 the dht and connection setup, move it into the Network setup with the tor and other network stuff info, not in main window that makes NEW USERS TO WORRY because the button is yellow or red and seems something bad... and why it happens they wonder, and they want it green... etc etc... 
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE

3. DHT stats.... ? do really somebody used this ever once? it should be in stats...
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE

4. Notification area... this area should be redone in another way... at least for me could be nice to have it somewhere and to be able to see hashing process, to be able to stop hashing process... but if we leave it there, at least removing the rest we have more space for it.
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE

5. Normal mode etc... HIDE IT, make this choices in setup... or make it visible only after enabling it in advanced setup, for new users this is just a mess and something to click and make RS unusable, and MAKER POSSIBLE TO SAVE THE STATE tired to change to not anon all the time....
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE

6. The other last 2 icons...
SHOULD BE OPTIONAL IN OPTIONS > APPEARENCE
![dsg](https://cloud.githubusercontent.com/assets/1285303/21582996/f60b30d0-d06c-11e6-86d5-e8890e21700d.png)

**Comment by felisucoibi** [comment]
I close this because is here: https://github.com/RetroShare/RetroShare/issues/640
